ext/sodium/php_libsodium.h

```diff
@@ -0,0 +1,109 @@
+
+#ifndef PHP_LIBSODIUM_H
+#define PHP_LIBSODIUM_H
+
+extern zend_module_entry libsodium_module_entry;
+#define phpext_libsodium_ptr &libsodium_module_entry
+
+#define PHP_LIBSODIUM_VERSION "2.0.0"
```

Comment:
Do you want to keep it sync with pecl for older version ?
Else, could make sense to drop specific version (and use PHP version as for most of other extensions)